modules/ding_ddbasic/modules/ding_ddbasic_opening_hours/plugins/content_types/weekly_opening_hours.inc

```diff
@@ -67,9 +74,9 @@ function ding_ddbasic_opening_hours_weekly_opening_hours_content_type_edit_form(
   );
 
   $options = array(
-    '_null' => t('Opening hours'),
+    DING_DDBASIC_OPENING_HOURS_NO_CATEGORY => t('Opening hours'),
```

Comment:
The DING_DDBASIC_OPENING_HOURS_NO_CATEGORY is a constant defined to 'no_category', so we're setting $options['no_category'] = t('Opening hours').

We do this to avoid using NULL as an array value, which will be cast to an empty string. If we used the empty string, Drupal wouldn't save it in the submit handler, because array_filter() would remove the empty value.